CLAUDE.md

```
@@ -0,0 +1,642 @@
+
+  ```bash
+  # Check pod status
+  kubectl get pods -n YOUR_NAMESPACE
+
+  # Check logs
+  kubectl logs -n YOUR_NAMESPACE
+
+  # Health check
+  curl http://localhost:8000/health
+  ```
+  ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)